fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/LuceneQueryComponent.java

```diff
@@ -93,11 +115,11 @@ public int planHash(@Nonnull final PlanHashKind hashKind) {
     @Override
     @Nonnull
     public String toString() {
-        return "LuceneQuery(" + query + ")";
+        return "LuceneQuery(" + comparison.getComparand() + ")";
     }
 
     @Override
     public int queryHash(@Nonnull final QueryHashKind hashKind) {
-        return HashUtils.queryHash(hashKind, BASE_HASH, query);
+        return HashUtils.queryHash(hashKind, BASE_HASH, comparison.getComparand());
```

Comment:
I don't understand this one. Can you please explain?

Reply:
I removed the `query` from this component as it just duplicates the lucene query. The query string itself lives in the `LuceneComparison` where it is represented as the comparand. That may not be ideal but it removes the redundancy from this component. So `comparison.getComparand()` is identical to the previous `query`.